emrun.py

```diff
@@ -1434,7 +1437,7 @@ def run():
         loge('Unable to find browser "' + str(options.browser) + '"! Check the correctness of the passed --browser=xxx parameter!')
         return 1
       browser_exe = browser[0]
-      browser_args = []
+      browser_args = shlex.split(options.browser_args)
```

Comment:
Perfect, let me merge this and adjust a couple of the lines below for `+=` instead of `=` so this won't get overwritten.